.landscaper/installation.yaml

```diff
@@ -0,0 +1,118 @@
+apiVersion: landscaper.gardener.cloud/v1alpha1
```

Comment:
as this is a example resource I suggest to make this clear and create a dedicated example resources folder

Reply:
We have removed this. Ingo plans the create some help documentation when we have modelled more components.